src/app/router.module.ts

```diff
@@ -1,9 +1,11 @@
 import { NgModule }             from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { HomeComponent } from './home/home.component';
+import { FaqComponent } from './faq/faq.component';
 
 const routes: Routes = [
-    { path: '', component: HomeComponent }
+    { path: '', component: HomeComponent },
+    { path: '', component: FaqComponent }
```

Comment:
So, path: property is the url where the Component is being displayed. In this case, if you make the path '/faq' and navigate to (local url)/faq, you'd see your component!